### PR TITLE
[Notifier] [Twilio] Add tests

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.1
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | ---
| License       | MIT
| Doc PR        | ---

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Twilio/TwilioTransportFactory.php [hunk 1]
```diff
@@ -32,6 +33,11 @@ public function create(Dsn $dsn): TransportInterface
         $accountSid = $this->getUser($dsn);
         $authToken = $this->getPassword($dsn);
         $from = $dsn->getOption('from');
+
+        if (!$from) {
+            throw new IncompleteDsnException('Missing from.', $dsn->getOriginalDsn());
+        }
+
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Twilio/TwilioTransportFactory.php:40]: This should be considered a bugfix

**Comment by plozmun** [comment]
Maybe you can add .gitattributes to exclude Test ?

**Comment by OskarStark** [comment]
@plozmun its already available 👍🏻

**Comment by plozmun** [comment]
> @plozmun its already available 👍🏻

True,  I thought It was not 🙈

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**